tests/test_nlg.py

```diff
@@ -79,3 +80,53 @@ def test_nlg_schema_validation_empty_buttons():
 def test_nlg_schema_validation_empty_image():
     content = {"text": "Hey there!", "image": None}
     assert CallbackNaturalLanguageGenerator.validate_response(content)
+
+
+@pytest.mark.parametrize("filled_slots", [
```

Comment:
@naoko Please also change this one to `pytest.mark.parametrize("slot_name, slot_value"`